pkg/localize/locales/en/cmd/kafka.en.toml

```
@@ -4,4 +4,28 @@ one = "kafka"
 
 [kafka.cmd.shortDescription]
 description = "Short description for command"
-one = "Create, view, use, and manage your Apache Kafka instances"
\ No newline at end of file
+one = "Create, view, use, and manage your Apache Kafka instances"
+
+[kafka.cmd.longDescription]
+description = "Long description for command"
+one = '''
+A collection of commands for managing and interacting with Apache Kafka instances.
+
+You may use `kafka [create/delete/list/describe]` to manage your Kafka instances, `kafka use` to control your currently utilised Kafka instance, and `kafka [topics/consumergroup]` to interact with Kafka instances.
```

```suggestion
Use `kafka [create/delete/list/describe]` to manage your Kafka instances, `kafka use` to control your current Kafka instance, and `kafka [topics/consumergroup]` to interact with you Kafka instances.
```
I made a couple edits to the text here, but I'm wondering if we even need this sentence. If someone runs this command, they'll see a list of the subcommands, which should make it obvious what they can do.

---

I have suggested it as form for separation between different set of commands, but after seeing your comment I think it might have been bad idea.